pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -144,15 +144,14 @@ func newEndpoint(e *model.NetworkEndpoint) (*endpoint.LbEndpoint, error) {
 
 // updateCluster is called from the event (or global cache invalidation) to update
 // the endpoints for the cluster.
-func updateCluster(clusterName string, edsCluster *EdsCluster) error {
+func (s *DiscoveryServer) updateCluster(clusterName string, edsCluster *EdsCluster) error {
 	// TODO: should we lock this as well ? Once we move to event-based it may not matter.
 	var hostname model.Hostname
-	var ports model.PortList
+	//var ports model.PortList
 	var labels model.LabelsCollection
 	var instances []*model.ServiceInstance
 	var err error
 
-	// This is a gross hack but Costin will insist on supporting everything from ancient Greece
 	if strings.Index(clusterName, "outbound") == 0 ||
```

Comment:
Now that we are in modern world,  you can also get rid of this entire if strings.index(clusterName..)..